### ISSUE TITLE
Feature request: Add an option under settings to disable default images selection

### ISSUE THREAD

**Issue by es20490446e** [AXeL-dev/easy-image-downloader#2]
In most circumstances you only want to download a single image from a website, so I think that having no image selected by default is better.

**Comment by AXeL-dev** [comment]
Indeed, this was the default behavior before i change it on one of the latest releases.

I guess this should be added as a switchable option under the settings, since in my use case which is Mangas scans scraping, i need the images to be selected by default, so i can just hit the download button to download them.

**Comment by es20490446e** [comment]
That sounds good to me 👍
